src/environment.js

```diff
@@ -1,7 +1,7 @@
 import { Environment, Network, RecordSource, Store } from 'relay-runtime';
 
 const fetchQuery = async (operation, variables, cacheConfig, uploadables) => {
-  const response = await fetch('https://graphql-sw-api-klmkpknwdl.now.sh/', {
+  const response = await fetch('https://graphql-sw-api-gxmdjgcfhi.now.sh/', {
```

Comment:
you can create an alias, so you don't need to update this every time you deploy to server \o/